guide/additional-info/changes-in-v13.md

```diff
@@ -0,0 +1,385 @@
+
+The Discord API now allows bots much more granular control over mention parsing, down to the specific ID if desired. Refer to the [Discord API documentation](https://discord.com/developers/docs/resources/channel#allowed-mentions-object) for more information.
+
+```diff
+- const client = new Discord.Client({ disableMentions: 'everyone' });
++ const client = new Discord.Client({ allowedMentions: { parse: ['users', 'roles'], repliedUser: true } });
+```
+
+:::
+
+### Replies / Message#reply
+
+`Message.reply()` will no longer result in the bot prepending a user mention to the content, replacing the behaviour with Discord's reply feature.
+
+`MessageOptions.reply` (User ID) has been removed, replaced with `MessageOptions.replyTo` (Message ID)
+
+```diff
+- channel.send("content", { reply: '123456789012345678' })` // User ID
++ channel.send("content", { replyTo: '765432109876543219' })` // Message ID
+```
+
+The new `MessageOptions.allowedMentions.repliedUser` boolean option determines if the reply will notify the author of the original message.
+
+```diff
+- message.reply('content')
++ message.reply('content', { allowedMentions: { repliedUser: false }})
+```
+
+### Bitfields / Permissions
+
+Bitfields such as Permissions and UserFlags will no longer support the use of string literals, instead requiring use of the flag.
+
+```diff
+- permissions.has('SEND_MESSAGES')
++ permissions.has(Permissions.FLAGS.SEND_MESSAGES)
+```
+
+### Webpack
+
+Webpack builds are no longer supported.
+
+## Changes and deletions
+
+### Client
+
+#### Client#emojis
+
+The Client Emoji manager is now a `BaseGuildEmojiManager`, providing cache resolution only and removing methods which would fail to create emojis as there was no Guild context.
+
+#### Client#generateInvite
+
+`Client#generateInvite` no longer supports `PermissionsResolvable` as its argument, requiring `InviteGenerationOptions`.
+
+To provide permissions, use `InviteGenerationOptions#permissions`.
+```diff
+- client.generateInvite([Permissions.FLAGS.SEND_MESSAGES]);
++ client.generateInvite({ permissions: [Permissions.FLAGS.SEND_MESSAGES] })
+```
+
+### ClientOptions
+
+#### ClientOptions#fetchAllMembers
+
+The `ClientOptions#fetchAllMembers` option has been removed.
+
+With the introduction of gateway intents, the `fetchAllMembers` Client option would often fail and causes major delays in ready states or even cause timeout errors. As its purpose is contradictory to Discord's intentions to reduce scraping of user and presence data, it has been removed.
+
+#### ClientOptions#messageEditHistoryMaxSize
+
+The `ClientOptions#messageEditHistoryMaxSize` option has been removed.
+
+To reduce on caching, discord.js will no longer store an edit history. This will need to be manually implemented if required.
+### Guild
+
+#### Guild#member
+
+The helper/shortcut method `Guild#member()` has been removed.
+
+```diff
+- guild.member(user);
++ guild.members.cache.get(user.id)
```

Comment:
```suggestion
+ guild.members.resolve(user);
```

Reply:
We don't encourage the user of `resolve` methods, so no change here imo